### DISCUSSION TITLE
How-to Update google home 1.10.0

### DISCUSSION THREAD

**Discussion by airxwing** [leikoilja/ha-google-home#725] [Q&A]
Hello, currently running version 1.9.17 via HACS. How can I update to the latest 1.10.0?
I am trying to resolve issue with the google authentication and going step by step. So, update first.
If don't mind I will appreciate detail steps to follow :)
Thank you!

**Comment by leikoilja** [comment]
Go to the HACS store (https://github.com/leikoilja/ha-google-home#hacs-installation)
<img width="1156" alt="image" src="https://github.com/leikoilja/ha-google-home/assets/10655107/ad0824a6-b5e3-4f20-81d9-a6c2ac9b2286">

**Reply by airxwing** [reply]
Thank you! I did not get the update notification, not sure why but what I did was to uninstall/remove and reinstall the newest version.
